src/kp2bw/convert.py

```
@@ -255,7 +295,7 @@ def _add_bw_entry_to_entries_dict(
         # Build FIDO2/passkey credentials from KeePassXC attributes
         fido2_credentials = self._build_fido2_credentials(entry)
         if fido2_credentials:
-            logger.info(f"  Migrating passkey for entry: {entry.title}")
+            logger.log(VERBOSE, f"  Migrating passkey for entry: {entry.title}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)